plugins/inputs/jolokia2/jolokia2_agent/README.md

```diff
@@ -1,10 +1,12 @@
-# Jolokia2 Proxy input plugin
+# Jolokia2 Proxy Input Plugin
```

Comment:
This is the jolokia2 agent readme file but it's talking about the jolokai2 proxy plugin. Not sure how that happened.

Reply:
Well the main issue is the non-standard structure of the Jolokia2 plugin(s). If you agree I will (in a separate PR) move `plugins/inputs/jolokia2/common` to `plugins/common/jolokia2` and flatten the structure to make `_agent` and `_proxy` visible under `plugins/inputs`... What do you think?

Reply:
That sounds good. Having them both directly in plugins/inputs will be much better. Thanks